test/qxelarator/test_Qxelarator.py

```diff
@@ -0,0 +1,65 @@
+import os
+import qxelarator
+import unittest
+
+
+class QxelaratorTest(unittest.TestCase):
+    def test_execute_file(self):
+        cqasm_file_name = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'bell_pair.qasm')
+        simulation_result = qxelarator.execute_file(cqasm_file_name, iterations=23)
+
+        self.assertIsInstance(simulation_result, qxelarator.SimulationResult)
+        self.assertEqual(simulation_result.shots_requested, 23)
+        self.assertEqual(simulation_result.shots_done, 23)
+        self.assertEqual(simulation_result.results, {"00": 23})
+        self.assertEqual(simulation_result.state, {"11": complex(1., 0.)})
+
+    def test_execute_string_fails_returns_simulation_error(self):
```

Comment:
This is not a simulation error, right? It is a parsing error due to the `,` after `X q`.

Reply:
That's right. But it tests the path where a simulation error is returned.

Reply:
We've got an answer to this from Pablo:

> The JSON output was useful when there was no API from python to retrieve the different pieces of data in the results.
> It was a request from QI but then I created the new API and they have integrated it.
Maarten had confirmed that they no longer use the JSON and that they were happy with the new API.
> It’s no longer useful as there is now possibility to get everything directly from Python.
> This should be in the docs